.github/workflows/infrastructure.yaml

```diff
@@ -28,23 +28,24 @@ jobs:
       matrix:
         stack: [infra]
         environment:
-          {% for env in matrix_envs -%}
+          {%- for env in matrix_envs %}
           - @{{ env.name }}
-          {% endfor %}
-    uses: tx-pts-dai/github-workflows/.github/workflows/tf-plan.yaml@v0.24.0
+          {%- endfor %}
+    uses: tx-pts-dai/github-workflows/.github/workflows/tf-plan.yaml@v1
     with:
       environment: ${{ matrix.stack }}-${{ matrix.environment }}
 
   apply:
     if: github.ref_name == github.event.repository.default_branch && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
Not required with max-parallel: 1

Reply:
it will run all jobs if set to false even if any fail.

https://github.com/swibrow/actions-playground/actions/runs/10844112319

Reply:
@swibrow fantastic, you broke my understanding of it. Basically with `max-parallel: 1` even though jobs aren't executed, they are in-progress.